Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/main/MainPresenter.java

```diff
@@ -135,6 +136,10 @@ public boolean handleOptionSelection(int itemId, ComponentActivity activity)
         new AfterDirectoryInitializationRunner().runWithLifecycle(activity,
                 () -> mView.launchOpenFileActivity(REQUEST_NAND_BIN_FILE));
         return true;
+
+      case R.id.menu_about:
+        new AfterDirectoryInitializationRunner().runWithLifecycle(activity,
+                this::showAboutDialog);
```

Comment:
Does the about dialog actually need directory initialization for anything?

Reply:
Oh woops, no